### DISCUSSION TITLE
How to add task to project scratch pad

### DISCUSSION THREAD

**Discussion by Imperium-Aneesh** [bramses/bramses-highly-opinionated-vault-2023#7] [Q&A]
In the test project, there is a task in the scratchpad but I am not able to figure out how to add tasks to the scratchpad using quick to scratchpad

**Comment by bramses** [comment]
I usually just add tasks to Kanban, but if you want to change the `add to scratchpad` template, you can!  Just change the capture format in Settings > Quick Add > Manage Macros > Add to Scratchpad > Capture Value

![Screenshot 2023-01-27 13-22-15](https://user-images.githubusercontent.com/3282661/215164873-a4bf28f6-bb7d-496f-b43e-be1231ad9c7a.png)
![Screenshot 2023-01-27 13-22-24](https://user-images.githubusercontent.com/3282661/215164883-9583ac89-b66e-4b42-b753-8a65bb7b6ccd.png)
![Screenshot 2023-01-27 13-22-34](https://user-images.githubusercontent.com/3282661/215164884-d9c537db-957f-42f4-9348-30d89d9d54d4.png)
